go/mysql/flavor_test.go

```diff
@@ -172,6 +187,11 @@ func TestGetFlavor(t *testing.T) {
 			capability: capabilities.CheckConstraintsCapability,
 			isCapable:  true,
 		},
+		{
+			version:    "8.0.20-log",
+			capability: capabilities.CheckConstraintsCapability,
+			isCapable:  true,
+		},
```

Comment:
Here, I think we could just add: 
```
		{
			version:    "",
			capability: capabilities.CheckConstraintsCapability,
			isCapable:  false,
		},
```
No?

Reply:
Added.